README.md

```diff
@@ -37,27 +42,31 @@ void main() {
 ```
 
 ## Conversations Whatsapp Sandbox
+
 To use the whatsapp sandbox you need to add `"ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX"` to the list of features you want enabled in `ApiConversationService` (named parameter `features` in the constructor).
 
 ## Generate documentation
+
 Documentation for this repository can be generated using the `dartdoc` command. After running the command, the documentation files can be found in `doc/api`. More information about dartdoc can be found in the official dartdoc [documentation](https://dart.dev/tools/dartdoc).
 
 ## Tests
+
 This repository includes unit tests, located in the `test` directory. To run them, execute `pub run test` in the command line. Covered are, per service, service tests and json (de)serialization tests of data classes.
 
-Before running the tests, create a `keys.json` file in `test_resources/` with the following content:
-```json
-{
-    "test": "YOUR_API_TEST_KEY",
-    "live": "YOUR_API_LIVE_KEY",
-    "msisdn": "YOUR_MSISDN"
-}
+Before running the tests, make sure you have the following environment variables in your system:
+
+```
+"API_TEST_KEY": "YOUR_API_TEST_KEY",
+"API_LIVE_KEY": "YOUR_API_LIVE_KEY",
+"MSISDN": "YOUR_MSISDN"
```

Comment:
What do you think about having an example for the msisdn? might not be super obvious for everyone and numbers can come in a few different formats.

Reply:
Sure! That could be incorporated in the readme. Maybe a table providing some an explanation and some examples for these values.